guides/common/modules/con_managing-multiple-content-view-environments.adoc

```diff
@@ -0,0 +1,10 @@
+[id="managing-multiple-content-view-environments_{context}"]
+= Managing multiple content view environments
+
+You can manage multiple content views in different environments by organizing, promoting, and maintaining them effectively.
+
+You can also assign multiple content view environments to hosts, both with and without Subscription Manager as well as view content view environment assignments, understand environment ordering and priority, and compare multiple content view environments with composite content views.
```

Comment:
question: does that mean you can consume content without `subscription-manager`? Or does that mean you can change the configuration with or without using `subscription-manager`?

Reply:
Good question :) I think the intent here should be to communicate that you can assign multiple content view environments either from the server (using hammer) or from the host (using subscription-manager.)

Reply:
I will rewrite this to incorporate what Jeremy says here. After I commit this, take a look and let me know if it ends up making sense and is accurate.